scripts/lint.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+echo "Running pyup_dirs..."
+pyup_dirs --py38-plus --recursive app tests
+
+echo "Running ruff..."
+ruff app tests --fix
+
+echo "Running isort..."
```

Comment:
ruff will handle sorting of import, please remove this

Reply:
Isort removed